src/bootstrap/src/core/build_steps/format.rs

```diff
@@ -24,20 +25,23 @@ fn rustfmt(src: &Path, rustfmt: &Path, paths: &[PathBuf], check: bool) -> impl F
     cmd.args(paths);
     let cmd_debug = format!("{cmd:?}");
     let mut cmd = cmd.spawn().expect("running rustfmt");
-    // Poor man's async: return a closure that'll wait for rustfmt's completion.
+    // Poor man's async: return a closure that might for rustfmt's completion (depending on the
```

Comment:
I assume this should read as the following?

```suggestion
    // Poor man's async: return a closure that might wait for rustfmt's completion (depending on the
```

Reply:
Fixed, thanks!